docs/snippets/common/button-story-docspage-with-mdx.js.mdx

```diff
@@ -14,15 +15,23 @@ export default {
   parameters: {
     docs: {
       page: CustomMDXDocumentation,
-    }
+    },
   },
 };
 
-export const Primary = () => <Button backgroundColor="primary" />;
+export const Primary = {
```

Comment:
Or this, I'm not sure which is better:

```js
export const Primary = {
  args: {
    backgroundColor: 'primary'
  }
};
```

Reply:
I've updated it to use args instead with the latest commit. Marking this as solved.